frontend/src/pages/TrainingReportForm/pages/__tests__/completeEvent.js

```diff
@@ -290,8 +290,8 @@ describe('completeEvent', () => {
       expect(status).toBeInTheDocument();
 
       // and the submit button is disabled
-      const submitButton = await screen.findByRole('button', { name: /submit/i });
-      expect(submitButton).toBeDisabled();
+      const submitButton = screen.queryByRole('button', { name: /submit/i });
```

Comment:
Does this need an await

Reply:
user-testing-library has a confusing array of promise and non-promise based selectors. "queryByRole" is synchronous.